syft/execution/communication.py

```diff
@@ -17,8 +15,10 @@ class CommunicationAction(Action):
     def __init__(
         self,
         obj_id: Union[str, int],
+        name: str,  # @TODO: maybe refactor into enum later?
         source: Union[str, int],
         destinations: List[Union[str, int]],
+        args_: Tuple,  # The arguments to the method call, needed for .share()
         kwargs_: dict,
     ):
         """Initialize an communication action
```

Comment:
I think I should have done it earlier sorry, but could you add a few lines to explain what represents each attribute for a CommunicationAction?